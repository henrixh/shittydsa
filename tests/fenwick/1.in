20 6
+ 3 34
? 4
+ 0 17
? 8
+ 9 11
? 20
