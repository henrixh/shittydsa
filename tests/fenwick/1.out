34
51
62
